.github/workflows/stubtest_third_party.yml

```
@@ -5,6 +5,8 @@ on:
     paths-ignore:
       - '**/*.md'
       - 'scripts/**'
+      # When requirements.txt changes, we run `daily.yml`, which includes third-party stubtest
+      - 'requirements-tests.txt'
```

Why don't we just do this, now that (following #8923), this job now has its own workflow file?

```suggestion
    paths:
      - 'stubs/**'
      - '.github/workflows/stubtest_third_party.yml'
      - 'tests/**'
      # When requirements.txt changes, we run `daily.yml`, which includes third-party stubtest
```